ext/surro-gate/extconf.rb

```diff
@@ -1,4 +1,8 @@
 require 'mkmf'
 
 have_header 'sys/epoll.h'
-create_makefile 'surro-gate/selector_ext'
+if RUBY_PLATFORM =~ /linux/ && !defined?(JRUBY_VERSION) && !ENV['SURRO_GATE_NOEXT']
+  create_makefile 'surro-gate/selector_ext'
+else
+  IO.write("Makefile", "all:\n")
```

Comment:
https://ruby-doc.org/stdlib-2.5.1/libdoc/mkmf/rdoc/MakeMakefile.html#method-i-dummy_makefile

Reply:
took a while to figure out how to use that method